### PR TITLE
To-Do List App Bundled with Webpack

### PR DESCRIPTION
**Pull request by johnkioko**
### I have implemented the following features:

- Created an index.html file with HTML markup. Create an empty div as a To Do List placeholder . 
- Set The index.html file as a template using the HTML Webpack Plugin.
- 
- Created an index.js file and set an array of some simple to do tasks (array of objects). Each task object contains three keys:
- description [string].
- completed [bool].
- index: [number].
- 
- Tasks are displayed dynamically from the JavaScript file.
- On page load render the dynamically created list of tasks in the dedicated placeholder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,37 @@
...
+- CSS
+- Git and Github
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
-
+[Live Demo Link]()
 
 ## Getting Started
```
Comment (omar-muhamad) [inline README.md:19]: - [ ] For the `Getting Started` section, Please add the instruction on how to clone your repo, install dependencies libraries `npm install`, run your build `npm run build`, and run your dev server `npm run start`. To make your readme file professional and user friendly.

File: README.md [hunk 2]
```diff
@@ -1,56 +1,37 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Basic Webpack Setup
 
+> This is a basic webpack setup project demo.
```
Comment (omar-muhamad) [inline README.md]: - [ ] Please add a proper project name and description to make your readme file more professional. 
For example, you can name the project `To-do list`.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @johnkioko,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @johnkioko 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**